.dev/utils/build.sh

```diff
@@ -83,33 +95,62 @@ if [ "${FRESH}" = true ]; then
 fi
 
 # Compile
-if [ "${DEBUG}" = true ]; then
-  ln -s -f $(pwd)/${RUNID}.R $(pwd)/${RUNID}_DEBUG.R
-  RUNID="${RUNID}_DEBUG"
-  git apply ${GISS_HOME}/.dev/utils/DEBUGGING_FLAGS.patch
-  make -j setup RUN=${RUNID} F90=mpif90 GC=${GC} MP=${OPENMP} MPI=YES MECH=carbon \
-    TYPE=Debug DEBUG=YES COMPILE_WITH_TRAPS=YES TRACEBACK=YES OVERWRITE=YES
-  git apply -R ${GISS_HOME}/.dev/utils/DEBUGGING_FLAGS.patch
+if [ "${CLASSIC}" = true ]; then
+  # TODO: GCCLASSIC_RUNDIR needs setting up first time?
```

Comment:
This will be required to fix the CI failure https://github.com/fetch4/GISS-GC/actions/runs/12281701044/job/34271167487?pr=67.